x-pack/docs/en/security/securing-communications/security-minimal-setup.asciidoc

```diff
@@ -24,8 +24,7 @@ free trial license to access all features of the Elastic Stack. See https://www.
 
 ==== Enable {es} security features
 
-When you use the basic license, the {es} security features are disabled by
-default. Enabling the {es} security features enables basic authentication so
+Enabling the {es} security features enables basic authentication so
```

Comment:
Authentication (xpack.security.enabled)  _is_ enabled for everyone no matter whether auto configuration runs or not . Maybe we can make this obvious and rework this entire page to reflect this ?

Reply:
Yeah, a lot in this page needs to be reworked.